pdi-weka-scoring-plugin/pom.xml

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>pentaho.weka</groupId>
+    <artifactId>pentaho-data-mining</artifactId>
+    <version>8.0-SNAPSHOT</version>
+  </parent>
+
+  <groupId>pentaho</groupId>
+  <artifactId>pdi-wekascoring-plugin-parent</artifactId>
+  <version>8.0-SNAPSHOT</version>
+  <packaging>pom</packaging>
+
+  <name>Pentaho Community Edition Project: pdi-wekascoring-plugin-parent</name>
+  <description>a Pentaho sponsored open source project</description>
+  <url>http://www.pentaho.org</url>
+
+  <modules>
+    <module>core</module>
+    <module>assemblies</module>
+  </modules>
+
+  <properties>
+    <license.licenseName>gpl_v2</license.licenseName>
+    <org.eclipse.swt.gtk.linux.x86_64.version>4.6</org.eclipse.swt.gtk.linux.x86_64.version>
+    <commons-vfs2.version>2.1-20150824</commons-vfs2.version>
+  </properties>
+
+  <build>
+    <resources>
+      <resource>
+        <directory>src/main/resources</directory>
+        <filtering>false</filtering>
+      </resource>
+      <resource>
+        <directory>src/main/resources-filtered</directory>
+        <filtering>true</filtering>
```

Comment:
the parent poms already take care of filtered resources, please remove this block
https://github.com/pentaho/maven-parent-poms/blob/master/pom.xml#L435